examples/json/server/main.go

```diff
@@ -83,6 +83,7 @@ func main() {
 			tch.NewInbound(channel, tch.ListenAddr(":28941")),
 			http.NewInbound(":24034"),
 		},
+		Interceptors: yarpc.Interceptors{requestLogInterceptor{}},
```

Comment:
If the yarpc.Interceptors wrapper will be added to any user interceptor, why not do it transitively. Can save one step for users. Same apply to filter.